### ISSUE TITLE
does not let you register if no remote key in .tickbinrc

### ISSUE THREAD

**Issue by jonotron** [tickbin/tickbin#286]
if you completely remove the `remote` line from `.tickbinrc` and you try to run `tick register` you get the following error:

```
You are already logged in, type tick logout to logout
```

The work around is to add the following line to your `.tickbinrc` file:

```
remote=
```

**Comment by chrisfosterelli** [comment]
@jonotron Was this on your development installation? If so, when you remove the `remote` flag RC will go find the "nearest" value for it, which will be the one in your home directory (`~/.tickbinrc`). That is why it still considers you logged in, because from tickbin's perspective `conf.remote` still has a value. When you set `remote=` it specifically sets it to an empty string so RC stops looking. 

So technically this is `rc` functioning as expected... but possibly a usability problem.

Related: I did discover that if you actually have no definition for `remote` the application does really fail on `tick register` with `Parameter "url" must be a string, not undefined`. This was introduced [here](https://github.com/tickbin/tickbin/pull/260/files#diff-a34ab6e935e2bdac68cdb4c2ec977d6aR16) and instead I think we need something like `conf.remote || ''`

**Comment by jonotron** [comment]
Ahhh... yes this makes sense. Thanks for clarifying.

Yes, let's leave this open but fix soon.

**Comment by jonotron** [comment]
I think this would block new users from registering

**Comment by chrisfosterelli** [comment]
Yeah, that's right. Will need to fix before anyone can register (with a
beta key).

**Comment by chadfawcett** [comment]
Merged into master in #302
